temporalio/client.py

```diff
@@ -1108,6 +1108,46 @@ async def get_worker_task_reachability(
             )
         )
 
+    async def describe_task_queue(
+        self,
+        task_queue: str,
+        task_queue_types: Sequence[TaskQueueType] = [],
+        report_pollers: bool = False,
+        report_stats: bool = False,
+        rpc_metadata: Mapping[str, str] = {},
+        rpc_timeout: Optional[timedelta] = None,
+    ) -> TaskQueueDescription:
+        """Describe task queue.
+
+        .. note::
+           This is only for unversioned workers. Worker versioning is not yet
+           supported for describing task queue.
+
+        Args:
+            task_queue: Name of the task queue. Sticky queues are not supported.
+            task_queue_types: Task queue types to report info about. If not
+                present or empty, all types are considered.
+            report_pollers: Include list of pollers for requested task queue types.
+            report_stats: Include task queue stats for requested task queue types.
+            rpc_metadata: Headers used on each RPC call. Keys here override
+                client-level RPC metadata keys.
+            rpc_timeout: Optional RPC deadline to set for each RPC call.
+        """
+        if not report_pollers and not report_stats:
+            raise ValueError(
+                "At least one of report_pollers or report_stats must be True"
+            )
```

Comment:
nit: Oh I see, so one is required.  I assumed not since they were both optional, and there was a default set of info.
If you want, you could avoid this friction, and runtime errors, by having a required enum field for either/both.